### ISSUE TITLE
Since fix for #479, paste doesn't work on some terminal anymore

### ISSUE THREAD

**Issue by jbonofre** [jline/jline3#514]
See https://issues.apache.org/jira/browse/KARAF-6613 for details.

**Comment by majp** [comment]
We also suffer from this in our software.
Last version that did not have this issue is 3.12.1
A fix would be great! Thanks in advance.

**Comment by jbonofre** [comment]
I fixed the issue be reverting the `beginPaste()` method in `LineReaderImpl`:

```
public boolean beginPaste() {
        final Object SELF_INSERT = new Object();
        final Object END_PASTE = new Object();
        KeyMap<Object> keyMap = new KeyMap<>();
        keyMap.setUnicode(SELF_INSERT);
        keyMap.setNomatch(SELF_INSERT);
        keyMap.setAmbiguousTimeout(0);
        keyMap.bind(END_PASTE, BRACKETED_PASTE_END);
        StringBuilder sb = new StringBuilder();
        while (true) {
            Object b = doReadBinding(keyMap, null);
            if (b == END_PASTE) {
                break;
            }
            String s = getLastBinding();
            if ("\r".equals(s)) {
            s = "\n";
            }
            sb.append(s);
        }
        regionActive = RegionType.PASTE;
        regionMark = getBuffer().cursor();
        getBuffer().write(sb);
        return true;
    }
```
I'm checking for a better fix, leveraging the change introduced in #479
